README.md

```diff
@@ -150,7 +149,7 @@ interface and the most expensive operation in CPU profiling is the interface
 method which in turn calls into runtime.assertI2T.  We need generics.
 
 #### Immutable B Tree
-A btree based on two principals, immutablability and concurrency. 
+A btree based on two principals, immutability and concurrency. 
```

Comment:
s/principals/principles
:)

Reply:
Good catch. I missed that. I will correct it as well.

Reply:
I'll re-approve immediately.